src/components/select/Select.tsx

```diff
@@ -103,49 +111,51 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
             },
           })}
         />
-        <FloatingPortal>
-          {open && (
-            <FloatingFocusManager context={context} visuallyHiddenDismiss initialFocus={-1}>
-              <div
-                {...getFloatingProps({
-                  ref: refs.setFloating,
-                  style: {
-                    ...floatingStyles,
-                    overflowY: 'auto',
-                    zIndex: 1000,
-                  },
-                })}
-              >
-                <Box border={1} radius={1} p={1} backgroundColor="white.main">
-                  {items.length > 0 ? (
-                    items.map((item, index) => (
-                      <SelectItem
-                        key={item.value}
-                        {...getItemProps({
-                          key: item.value,
-                          ref(node) {
-                            listRef.current[index] = node;
-                          },
-                          onClick() {
-                            handleSelectItem(item);
-                            refs.domReference.current?.focus();
-                          },
-                        })}
-                        active={activeIndex === index || value === item.value}
-                      >
-                        {item.label}
-                      </SelectItem>
-                    ))
-                  ) : (
-                    <Typography variant="text12" color="neutral.400" p={1}>
-                      {noResults}
-                    </Typography>
-                  )}
-                </Box>
-              </div>
-            </FloatingFocusManager>
-          )}
-        </FloatingPortal>
+        {!disabled && (
```

Comment:
Instead of conditionally rendering content, I'd explore the possibility to do something with the floating-ui library, which will be maybe more conscious with accessibility. I was thinking in:
```typescript
const click = useClick(context, { enabled: !disabled });
```

Reply:
Yes, good catch. I think I would add this `useClick` change anyway. For it would be okay if you conditionally render but at the same level as the `open` conditional render:
```typescript
{open && !disabled &&
    <FloatingFocusManager context={context} visuallyHiddenDismiss initialFocus={-
        <div
```